akka-docs/src/test/scala/docs/stream/operators/Map.scala

```diff
@@ -0,0 +1,17 @@
+package docs.stream.operators
+
+//#imports
+import akka.NotUsed
+import akka.stream.scaladsl._
+
+//#imports
+
+object Map {
+
+  //#map
+  def mapExample: Source[String, NotUsed] = {
+    val source = Source(1 to 10)
+    source.map(elem => elem.toString)
```

Comment:
Might be nice to add the types, to show that this `map` changes the `Source[Int]` into a `Source[String]`?

Reply:
Added the type to the source before applying the map.